virttest/qemu_vm.py

```diff
@@ -2083,7 +2083,16 @@ def add_disable_modern(devices, dev, dev_type):
                               "virtio-serial-pci", "virtio-rng-pci"]
         for device in devices:
             dev_type = device.get_param("driver")
-            if dev_type in virtio_pci_devices:
+            host_kernel_ver = process.system_output("uname -r",
+                                                    shell=True,
+                                                    verbose=False)
+            # Currently only need to add the options for RHEL.7.2 host,
+            # you can add them for other hosts if necessary
+            if "3.10.0-327." in host_kernel_ver:
```

Comment:
The "-327" is only introduced and used by redhat release.
Can you distinguish it by other way? for example, qemu version?